packages/common/tests/utils.spec.ts

```diff
@@ -0,0 +1,89 @@
+import * as tape from 'tape'
+
+import { Common } from '../src/common'
+import { parseGethGenesis } from '../src/utils'
+
+tape('[Utils/Parse]', (t) => {
+  t.test('should parse geth params file', async (t) => {
+    const json = require(`../../client/test/testdata/geth-genesis/testnet.json`)
```

Comment:
Hmm. Won't make it a blocker here but this is a bit dangerous.

So if someone reorganizes test data in client, this will (at least locally) remain unnoticed when someone is working "client only" and running the client tests and the like. Will break on CI then so somewhat ok, but nevertheless we should generally avoid and in doubt rather take over.

At some point we might also want to consolidate testdata a bit more (or at least discuss), and do a dedicated directory `testdata` on `packages` folder level or the like and in turn keep the test data a bit more curated and high quality level. Just some thoughts for now though.